libpng.spec

```diff
@@ -1,5 +1,6 @@
 ### RPM external libpng 1.2.47
 Source: http://downloads.sourceforge.net/libpng/libpng12/%realversion/%n-%realversion.tar.bz2
+Provides: libpng15.so.15()(64bit) libpng15.so.15(PNG15_0)(64bit)
```

Comment:
@amaltaro , this will fail if you run on a system where libpng version is different. As I mentioned, matplotlib has used libpng from system that is why cmsBuild complain about the `libpng15.so.15()(64bit)`  dependency. Please fix the matplotlib spec

Reply:
Yes, I forgot to remove that line from the libpng spec. I'm trying the matplotlib update in a different PR (hopefully I won't need to bug you :))